projects/SEP_TEST/eligibility/SEP_TEST_eligibility_v1.json

```diff
@@ -0,0 +1,82 @@
+[
+    {
+        "field_name":"sep_eligibility_field_1",
+        "form_name":"sep_eligibility",
+        "section_header":"",
+        "field_type":"text",
+        "field_label":"How old are you?",
+        "select_choices_or_calculations":"",
+        "field_note":"",
+        "text_validation_type_or_show_slider_number":"number",
```

Comment:
Should we use `integer` here, as the 'integer' validation only allows whole numbers, while the `number` validation allows numbers with decimals?

Reply:
Yes good point.